examples/vmi-file-memory-backing.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-file-backing-memory
+  name: vmi-file-backing-memory
+spec:
+  domain:
+    devices: {}
+    memory:
+      fileBacked: {}
```

Comment:
Hey.
What about an approach like
```
memory:
  backend:
    file: {}
```

Instead, just because we don#t know what options we might need in future.
And if other backends appear

Reply:
Cool!
I like that. T

Reply:
Done.